src/Service/HealthCheck/HealthCheckHelper.cs

```diff
@@ -0,0 +1,255 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using Azure.DataApiBuilder.Config.HealthCheck;
+using Azure.DataApiBuilder.Config.ObjectModel;
+using Azure.DataApiBuilder.Product;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+
+namespace Azure.DataApiBuilder.Service.HealthCheck
+{
+    /// <summary>
+    /// Creates a JSON response for the health check endpoint using the provided health report.
```

Comment:
Class should be for similar objects with properties and methods. The description here suggests what the class does - which should actually be a description of one of the class methods, not of the class itself.